### ISSUE TITLE
GCP-AppEngine-Flexible: googleapis-common-protos==1.4.0 leads to from google.appengine.api import error

### ISSUE THREAD

**Issue by yaalaa** [googleapis/googleapis#189]
It all happens on GCP AppEngine Flexible Environment Python VM.
 
It seems `googleapis-common-protos==1.4.0` installs something into `google/appengine` folder. Likely some logging-related classes.
I get `googleapis-common-protos` as indirect dependency installing `gcloud`.
Having `gcloud` with `googleapis-common-protos==1.4.0` I try to instantiate `google.datastore.Client` and get exception like that:
```
from google.appengine.api import app_identity
ImportError: No module named api
```

Having `gcloud` with `googleapis-common-protos==1.3.5` all works fine.

Please let me know this issue should be opened in another place.

**Comment by geigerj** [comment]
Sorry about that; `googleapis-common-protos==1.4.0` was broken on certain platforms due, as you suspected, to the inclusion of an appengine logging class. The full discussion is available [here](https://github.com/googleapis/packman/pull/147#issuecomment-257454233). This is fixed in `googleapis-common-protos==1.4.0`.

cc: @bjwatson

**Comment by bjwatson** [comment]
Correction, this is fixed in [`googleapis-common-protos==1.5.0`](https://pypi.python.org/pypi/googleapis-common-protos).
